### ISSUE TITLE
Changes in golang.org/x/tools/oracle break gotools/oracle

### ISSUE THREAD

**Issue by NHOrus** [visualfc/gotools#2]
Commit b28839e4bdc90af193b3e38f473e5f9175158b41 in said golang.org/x/tools/oracle introduced incompatible changes in API of tools/oracle

Before changes function signature was 
`func Query(args []string, mode, pos string, ptalog io.Writer, buildContext *build.Context, reflection bool)` and after them it's `func Run(q *Query) error` with `type Query` mapped to arguments of `func Query` as follow:
```
oracle.Query{
    Mode:            mode,
    Pos:             pos, //oraclePos
    Build:           buildContext, // &build.Default,
    Scope:           args,
    PTALog:          ptalog, //nil,
    Reflection:      reflection, //oracleReflect,
```

with Serial and WriteTo being applied to Query, not Result.

Path is to follow.

**Comment by NHOrus** [comment]
Also, fix for error `./oracle.go:33: cannot use runOracle (type func(*command.Command, []string)) as type func(*command.Command, []string) error in field value` would be also included

**Comment by NHOrus** [comment]
Made an error, fix is in pull request #3

**Comment by NHOrus** [comment]
Relevant thing was closed.
